packages/wonder-blocks-icon/src/index.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import Icon from "./components/icon.js";
-import icons from "./util/icon-assets.js";
+import * as icons from "./util/icon-assets.js";
```

Comment:
**suggestion:** This could be changed to:

```
export * as icons from "./util/icon-assets.js";
```

And then drop the additional export on line 7.

However, since this also exports the `__esModule` export, that's going to be confusing. Instead, how about...

```
import * as iconAssets from "./util/icon-assets.js";

const {__esModule, ...icons} = iconAssets;

export {icons};
```

Reply:
I love it!  I was trying to think of a way to do this this morning.  Thank you. 💚